src/v2/view-builder/views/captcha/CaptchaHelper.js

```diff
@@ -40,21 +25,28 @@ export function renderCaptcha(captchaConfig, form, onCaptchaSolvedCallback) {
   const captchaObject = captchaConfig.type === 'HCAPTCHA' ? hcaptcha : grecaptcha;
 
   // Iterate over all the primary buttons in the form and bind CAPTCHA to them
-  _.each(form.$('.button-primary'), (elem) => {
+  _.each(form.$('.o-form-button-bar .button[type=submit]'), (elem) => {
     const captchaId = captchaObject.render(elem, {
       sitekey: captchaConfig.siteKey,
-      callback: onCaptchaSolvedCallback
+      callback: (token) => {
+        // We reset the Captchas using the id(s) that were generated during their rendering.
+        const submitButtons = document.getElementsByClassName('button-primary');
```

Comment:
use same selector `'.o-form-button-bar .button[type=submit]'`

Reply:
ah good point, I'll update this